modules/custom-cert-new-install-roxctl.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * configuration/add-custom-certificates.adoc
+:_module-type: PROCEDURE
+[id="custom-cert-new-install-roxctl_{context}"]
+= Adding a custom certificate during a new installation with roxctl CLI
+
+If you are installing {product-title} using the `roxctl` CLI, provide the certificate and key files when you run the installation program.
+
+.Procedure
+
+* For the non-interactive installer, use the `--default-tls-cert` and   `--default-tls-key` options:
++
+[source,terminal]
+----
+$ roxctl central generate --default-tls-cert "cert.pem" --default-tls-key "key.pem"
+----
+* For the interactive installer, provide the certificate and key files when   you enter answers for the prompts:
++
+[source,text]
+----
+...
+Enter PEM cert bundle file (optional): <cert.pem>
```

Comment:
Its not clear that this is the full path of the cert or key or if its expected in your pwd here.